lexer/lexer.go

```diff
@@ -21,10 +21,15 @@ func New(input string) *Lexer {
 	l.readChar()
 	l.FSM = fsm.NewFSM(
 		"initial",
+		/*
+			Initial state is default state
+			Nosymbol state support we identify tok ':' is symbol or json
+			Method state support we identify tok 'class' is kind of keyword or Ident
```

Comment:
This should be changed to
`Method state helps us identify 'class' literal is a keyword or an identifier`